frontend/src/components/ItemCardList.tsx

```diff
@@ -49,7 +49,7 @@ const ItemCardList = ({
         ))}
       </div>
     ) : (
-      <p>No {title.toLowerCase()}.</p>
+      <p> Nothing to Display . </p>
```

Comment:
Again, please update this. There's no need for extra whitespaces before the phrase or before/after a dot.
Also, no need to have `display` capitalized - it's a regular sentence.
```suggestion
      <p>Nothing to display.</p>
```

Reply:
Ok , will make correction here 🙂

Reply:
fixed 👍